borgmatic/tests/unit/borg/test_hook.py

```diff
@@ -0,0 +1,10 @@
+from flexmock import flexmock
+
+from borgmatic.commands import hook as module
+
+
+def test_exec_cmd():
```

Comment:
Even though there's only a single test case, it'd be helpful to encode in the test name what exactly you're testing about the unit under test. Example: `test_exec_cmd_invokes_each_command()`.

Speaking of which, either this test case or another test case could test that it actually calls multiple different commands if there are indeed multiple in the list. Not strictly necessary though.